src/TrixiTest.jl

```diff
@@ -1,50 +1,9 @@
 module TrixiTest
 
-using Test: @test
+include("auxiliary.jl")
+include("macros.jl")
```

Comment:
```suggestion
using Test: @test
using TrixiBase: mpi_isroot, trixi_include

include("auxiliary.jl")
include("macros.jl")
```
Usually, we try to have every imports in the main entry file of a package. See also my suggestion below.